### ISSUE TITLE
Inline code with masked links

### ISSUE THREAD

**Issue by Eisenwave** [TCCPP/wiki-articles#17]
Various articles have the formatting `**[std::string](URL)**`. This was used because Discord did not render this Markdown correctly, at least not on mobile iirc.

I have checked recently, and now this appears correct on Mobile and on Desktop, so we should use inline code instead.

However, it looks like the wheatley parser appears bugged because links of the form:
```md
[`awoo`][ref]
```
... don't render for some reason.

In short, we should:

1. Resolve the bug with parsing inline code in tagged, masked, links.
2. Migrate the old links to used inline code.
